price-feeder/oracle/provider/binance.go

```diff
@@ -1,20 +1,23 @@
 package provider
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
+	"net/url"
 	"strings"
+	"sync"
 	"time"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gorilla/websocket"
+	"github.com/rs/zerolog"
 	"github.com/umee-network/umee/price-feeder/oracle/types"
 )
 
 const (
-	binanceBaseURL        = "https://api.binance.com"
-	binanceTickerEndpoint = "/api/v3/ticker/24hr"
+	binanceHost          = "stream.binance.com:9443"
+	binancePath          = "/ws/umeestream"
```

Comment:
Is `umeestream` the stream name? That can be anything, right?

Reply:
right, any stream name